addons/l10n_hk/models/res_bank.py

```diff
@@ -0,0 +1,56 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import single_email_re
+
+
+class ResPartnerBank(models.Model):
+    _inherit = 'res.partner.bank'
+
+    country_code = fields.Char(related='partner_id.country_code', string="Country Code")
+    l10n_hk_fps_type = fields.Selection([('id', "FPS ID"), ('mobile', "Mobile Number"), ('email', "Email Address")], string='FPS Type')
+    l10n_hk_fps_identifier = fields.Char(string="FPS ID/Mobile Number/Email Address")
```

Comment:
Just call it FPS ID. It'll be shorter and cleaner ; even when it contains a mail, this mail IS the FPS id.

Shouldn't those two fields be declared in account_qr_code_emv ? Is there that much flexibility in what can go into that field in other countries needing it ?

Reply:
Hmm, ok. We'll see if the field needs to be moved in the parent module when we add other modules depending on this type of QR code. 

For the label, I really don't agree: it's too long and it's not clear what should go into it because of this A/B/C style. Plus, if "FPS ID" is not clear enough, then the field should not be called l10n_hk_fps_identifier. 

Anyway, with the imminent freeze, I'll try some changes myself, so that we can move on faster :)

Reply:
Discussed with jco: let's just call that field "FPS Identifer" ; should be clear enough, and we can change that based on feedbacks later on if experience shows there is confusion among the users between this and "FPS ID" selection value.